tests/daemon/test-persistent-cache.c

```diff
@@ -878,24 +878,13 @@ static void
 test_persistent_cache_builds_boot_metadata_file (gboolean     *unused,
                                                  gconstpointer dontuseme)
 {
-  EmerPersistentCache *cache = make_testing_cache ();
-
-  GError *error = NULL;
-  gboolean get_succeeded =
-    emer_persistent_cache_get_boot_time_offset (cache, NULL, TRUE, &error);
-  g_assert_no_error (error);
-  g_assert_true (get_succeeded);
-
-  gint64 first_offset = read_offset ();
-
   gint64 absolute_time, relative_time;
   g_assert_true (emtr_util_get_current_time (CLOCK_BOOTTIME, &relative_time));
   g_assert_true (emtr_util_get_current_time (CLOCK_REALTIME, &absolute_time));
 
-  get_succeeded =
-    emer_persistent_cache_get_boot_time_offset (cache, NULL, TRUE, &error);
-  g_assert_no_error (error);
-  g_assert_true (get_succeeded);
+  EmerPersistentCache *cache = make_testing_cache ();
+
+  gint64 first_offset = read_offset ();
 
   g_assert_true (boot_timestamp_is_valid (relative_time, absolute_time));
   gint64 second_offset = read_offset();
```

Comment:
We should remove this line and replace `second_offset` with `0` below. The way this code is written now makes it seem like the `g_assert_cmpint` is testing that `boot_timestamp_is_valid` didn't change the boot offset, but `boot_timestamp_is_valid` is a testing function, so even if it could have such a bug, it wouldn't be worth testing for it.